source/ui/BioXAS/BioXASSidePersistentView.cpp

```diff
@@ -61,7 +72,7 @@ BioXASSidePersistentView::~BioXASSidePersistentView()
 void BioXASSidePersistentView::onCalibrateEnergyButtonClicked()
 {
 	bool inputOK = false;
-	double newEnergy = QInputDialog::getDouble(this, "Monochromator Energy Calibration", "Enter current calibrated energy:", BioXASSideBeamline::bioXAS()->mono()->energyControl()->value(), 0, 10000000, 1, &inputOK, Qt::Sheet);
+	double newEnergy = QInputDialog::getDouble(this, "Monochromator Energy Calibration", "Enter current calibrated energy:", BioXASSideBeamline::bioXAS()->mono()->energyControl()->value(), -10000000, 10000000, 1, &inputOK, Qt::Sheet);
```

Comment:
I think it may technically be possible at some point during a crystal change, though I'd have to run one though again to check.